packages/target-electron/src/ipc.ts

```diff
@@ -173,6 +175,41 @@ export async function init(cwd: string, logHandler: LogHandler) {
     ev.returnValue = app.getPath(arg)
   })
 
+  /**
+   * https://www.electronjs.org/docs/latest/api/system-preferences#systempreferencesgetmediaaccessstatusmediatype-windows-macos
+   */
+  ipcMain.handle('checkMediaAccess', (_ev, mediaType: MediaType) => {
+    if (!systemPreferences.getMediaAccessStatus) {
+      throw new Error('getMediaAccessStatus is not supported on this platform')
+    }
+    if (mediaType === 'camera') {
+      return systemPreferences.getMediaAccessStatus('camera')
+    } else if (mediaType === 'microphone') {
+      return systemPreferences.getMediaAccessStatus('microphone')
+    } else {
+      throw new Error('checkMediaAccess: unsupported media type')
+    }
+  })
+
+  /**
+   * https://www.electronjs.org/docs/latest/api/system-preferences#systempreferencesaskformediaaccessmediatype-macos
+   */
+  ipcMain.handle(
+    'askForMediaAccess',
+    (_ev, mediaType: MediaType): Promise<boolean> => {
+      if (!systemPreferences.askForMediaAccess) {
+        throw new Error('askForMediaAccess is not supported on this platform')
+      }
+      if (mediaType === 'camera') {
+        return systemPreferences.askForMediaAccess('camera')
```

Comment:
This is unused though, right? It'd be nice to remove it then, for security.

Reply:
I would prefer to keep it, otherwise we would have to change the runtime method signature to only support microphone which might not be sufficient for other use cases